core/store/src/test_utils.rs

```diff
@@ -49,7 +49,7 @@ pub fn test_populate_trie(
 }
 
 pub fn gen_accounts(rng: &mut impl Rng, max_size: usize) -> Vec<AccountId> {
-    let alphabet = &b"abcdefgh"[0..rng.gen_range(4, 8)];
```

Comment:
Is this intentional? In general, there are benefits in randomizing the size of the alphabet -- if the alphabet is large, it's hard to get repeates and dupes.

Reply:
Yes, I actually added this function and I think all of use cases of this function right now actually do not want repeats. I will make the name more obvious for that though.